app/src/main/java/com/github/wnder/GuessLocationActivity.java

```diff
@@ -51,6 +53,9 @@
     private GeoJsonSource guessSource;
     private ValueAnimator animator;
 
+    private String pictureID = "";
```

Comment:
We should maybe consider having a name for the default not yet initialised ID one day.